plugins/ircbridge/ircbridge.js

```diff
@@ -20,8 +33,20 @@ module.exports = function loadPlugin(resources, service) {
 handlers.enable = function(cb) {
     persist.load().then(function(result) {
         storage = result;
-        storage.telegramAliases = storage.telegramAliases || {};
-        storage.telegramColors = storage.telegramColors || {};
+        storage.telegramUserPreferences = storage.telegramUserPreferences || {};
+
+        // Load legacy config
+        if (storage.telegramColors) {
+            object.keys(storage.telegramColors).forEach(
+                user => setUserPreference(user, PREFERENCES_KEY_COLOUR, storage.telegramColors[user]));
+            storage.telegramColors = {};
+        }
+        if (storage.telegramAliases) {
+            object.keys(storage.telegramAliases).forEach(
```

Comment:
You could simply use `Object.keys(storage.telegramAliases || {})` and iterate that. It saves a couple of branches.

Also, forEach is 95% slower than just using `for..in`

Reply:
There's a reason lodash is in all of my projects, `_.forEach` is both fast and works directly on an object